modules/auth/server/procedures.ts

```
@@ -0,0 +1,82 @@
+      const cookies = await getCookies();
+      cookies.set({
+        name: AUTH_COOKIE,
+        value: data.token,
+        httpOnly: true,
+        path: "/"
+      });
+
+      return data;
+    }),
+  logout: baseProcedure
+    .mutation(async ({ ctx }) => {
```

ESLint

[error] 'ctx' is defined but never used.

(@typescript-eslint/no-unused-vars)